packages/eslint-plugin-pf-codemods/index.js

```diff
@@ -42,6 +42,8 @@ const rules = {
   "refactor-breakpointmods": require('./lib/rules/refactor-breakpointmods'),
   "datatoolbar-rename-toolbar": require('./lib/rules/datatoolbar-rename-toolbar'),
   "chartVoronoiContainer-remove-allowTooltip": require('./lib/rules/chartVoronoiContainer-remove-allowTooltip'),
+  "createContainer-insteadof-allowZoom": require('./lib/rules/createContainer-insteadof-allowZoom'),
+  "createContainer-insteadof-allowZoom": require('./lib/rules/createContainer-insteadof-allowZoom'),
```

Comment:
Only need this once. Can you also rename the rule `chart-remove-allowZoom`?

Reply:
DOne